### PR TITLE
Update leg sense organs

### PR DESCRIPTION
**Pull request by gouttegd**
Update sensory organs of legs based on new data from Kuan et al. (2020) (FBrf0247324).

Related to #1099

### CODE REVIEW

File: src/ontology/fbbt-edit.obo [hunk 1]
```diff
@@ -119244,6 +119252,412 @@ is_a: FBbt:00058039 ! metathoracic tibial campaniform sensillum
...
+def: "A stretch receptor neuron with a dendrite in the tibia-tarsus joint of the metathoracic leg." [FBC:DPG]
+subset: cur
+intersection_of: FBbt:00048300 ! adult leg stretch receptor neuron
+intersection_of: has_sensory_dendrite_in FBbt:00051088 ! metathoracic tibia-tarsus joint
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-30T10:42:11Z
+
+[Term]
+id: FBbt:00058100
+name: prothoracic coxal strand receptor
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:119652]: I would put 'neuron' on the end of this for clarity.

File: src/ontology/fbbt-edit.obo [hunk 2]
```diff
@@ -119244,6 +119252,412 @@ is_a: FBbt:00058039 ! metathoracic tibial campaniform sensillum
...
+def: "A neuron with a cell body in the ventral nerve cord and a sensory dendrite terminating at the cuticle of the prothoracic coxa. There is one such neuron per prothoracic leg. It may be analog to the strand receptor previously only reported in orthopteran insects (Kuan et al., 2020)." [FlyBase:FBrf0247324]
```
Comment (Clare72) [inline src/ontology/fbbt-edit.obo:119653]: 'It may be analog to' -> 'It may be analogous to' or 'It may be an analog to'

**Review by Clare72 — CHANGES_REQUESTED** [review]
mostly good, main thing is to change all the 'mechanosensory neuron of leg campaniform sensillum' terms (including the old ones) to use 'mechanosensory neuron'.

File: src/ontology/fbbt-edit.obo [hunk 3]
```diff
@@ -119244,6 +119252,412 @@ is_a: FBbt:00058039 ! metathoracic tibial campaniform sensillum
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-04-28T11:54:22Z
 
+[Term]
+id: FBbt:00058061
+name: mechanosensory neuron of tibial campaniform sensillum
+def: "Mechanosensory neuron with a dendrite in a campaniform sensillum of the tibia." [FlyBase:FBrf0231054]
+subset: cur
+intersection_of: FBbt:00048057 ! sensory neuron of campaniform sensillum
```
[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/fbbt-edit.obo:119260]: > if you don't mind doing this?

No problem.

**Review by Clare72 — APPROVED** [review]
I also edited the logical definition on the parent class 'mechanosensory neuron of leg campaniform sensillum'.
You can merge when checks pass

**Pull request merged**